UNRELEASED.md

```diff
@@ -6,6 +6,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- `Popover` and `PositionedOverlay` can now accept a `mutationObserveConfig` prop ([#4303](https://github.com/Shopify/polaris-react/pull/4303))
```

Comment:
```suggestion
- Added `mutationObserveConfig` prop to `Popover` and `PositionedOverlay` ([#4303](https://github.com/Shopify/polaris-react/pull/4303))
```

Per our [guidelines](https://github.com/Shopify/polaris-react/blob/main/documentation/Versioning%20and%20changelog.md#user-content-component-name-formats)

`Past tense verb + brief issue/enhancement description in <ComponentName>`